src/main/msp/msp.c

```diff
@@ -1845,6 +1845,10 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
 #if defined(USE_THROTTLE_BOOST)
         sbufWriteU8(dst, currentPidProfile->throttle_boost);
 #else
+
+#if defined(USE_TURN_THROTTLE_BOOST)
+        sbufWriteU8(dst, currentPidProfile->turn_throttle_boost_factor);
+#endif
```

Comment:
We usually don't add any new feature to the MSP. We usually wait for one version being CLI only before adding it. The MSP stuff needs to be backwards compatible and if after testing something needs to be changed, we need to maintain the "bad" version and add a "new" one.

Reply:
Removed!